internal/database/db.go

```diff
@@ -43,5 +43,18 @@ func New() *sql.DB {
 		panic(err)
 	}
 
+	if _, err := db.ExecContext(
+		ctx,
+		`CREATE TABLE account (
+			id          TEXT NOT NULL,
+			name        TEXT NOT NULL,
+			customer_id TEXT NOT NULL,
+
+			PRIMARY KEY (id)
```

Comment:
Will this "need" an index on `customer_id`?  IIRC the use case is to show the accounts that belong to the "current customer".

Reply:
Also, when we roughed out the UI, I'm pretty sure we intended to show balance next to each account.

Reply:
See https://github.com/dogmatiq/example/issues/20#issuecomment-554802647

Reply:
An index on customer sounds good. I don't remember doing a UI rough mock up? That would have helped.

Reply:
It was something we drew very quickly on the whiteboard.  I think you documented the columns you'd want in #20.

Reply:
Can sqlite add indexes as part of the `CREATE TABLE`? I can't seem to find any docs about doing it in there, only as a separate statement.

Is this ok?
```go
	if _, err := db.ExecContext(
		ctx,
		`CREATE TABLE account (
			id          TEXT NOT NULL,
			name        TEXT NOT NULL,
			customer_id TEXT NOT NULL,
			balance     INTEGER NOT NULL,

			PRIMARY KEY (id)
		);

		CREATE INDEX idx_account_customer ON account (customer_id);
		`,
	); err != nil {
		panic(err)
	}
```

Reply:
It [looks like](https://www.sqlite.org/syntax/table-constraint.html) you're correct, only primary keys, unique constraints, and check constraints can be specified inside the `CREATE TABLE` DDL.

Hrm, I know the MySQL driver doesn't allow you to specify multiple statements in a single `Exec()` call like that (by default), but if it works it's fine.  Infact, I guess if that works you could put the entire schema definition in a single string constant and exec that.

Reply:
Would that make it harder to track any error that gets panic, or much the same?

Reply:
It could make it a little harder if the error message doesn't include the relevant information about the failing query, though you're likely going to know what you changed if you do small commits :)